src/main/java/store/Application.java

```diff
@@ -1,7 +1,19 @@
 package store;
 
+import store.controller.StoreController;
+import store.domain.service.MemberShipCalculator;
+import store.domain.service.PromotionCalculator;
+import store.view.InputView;
+import store.view.OutputView;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        OutputView outputView = new OutputView();
+        InputView inputView = new InputView();
+        MemberShipCalculator memberShipCalculator = new MemberShipCalculator();
+        PromotionCalculator promotionCalculateService = new PromotionCalculator();
+        StoreController storeController = new StoreController(memberShipCalculator, promotionCalculateService, outputView, inputView);
```

Comment:
controller 내부에서 생성자 주입으로 이미 명시를 해두었는데, 객체 생성을 한번 더 하신 이유가 궁금해요!
InputView와 OutputView는 static으로 호출하고 아래처럼 하시면 어떨까요??

```suggestion
        StoreController storeController = new StoreController(new MemberShipCalculator(), new PromotionCalculateService());
```